azure-common-resources/modules.tf

```diff
@@ -46,6 +43,25 @@ module "create-network" {
   depends_on = [module.create-platform-prerequisite]
 }
 
+module "create-publicip" {
+  source = "./create-publicip"
+
+  count                   = var.create_publicip ? 1 : 0
+  cost_center             = var.cost_center
+  customer_name           = var.customer_name
+  project_name            = var.project_name
+  location                = var.location
+  project_stage           = var.project_stage
+  create_publicip         = var.create_publicip
+  create_dnsrecord        = var.create_dnsrecord
+  dns_record              = var.dns_record
+  dns_zone_name           = var.dns_zone_name
+  dns_zone_rg             = var.dns_zone_rg
+  publicip_resource_group = var.publicip_resource_group
```

Comment:
We should define a default value for this new var otherwise the 'classic' deployment flow will be broken or, at least it will need one more var...